spec/bolt/inventory_spec.rb

```diff
@@ -756,14 +756,17 @@ def common_data(transport)
         'alias' => %w[bar baz],
         'config' => { 'ssh' => { 'disconnect-timeout' => 100 } },
         'facts' => { 'foo' => 'bar' }
+      }, {
+        'name' => 'foo',
+        'alias' => %w[bak]
       }] }
     }
 
     let(:inventory) { Bolt::Inventory.new(data) }
-    let(:target) { get_target(inventory, 'foo') }
```

Comment:
Does `get_target` not work for this?

Reply:
No - I'm not entirely sure what's happening differently between `get_target` and `Inventory#get_targets`, but when a target shows up in another group it gets ignored and its alias doesn't get added to all of the groups that it's a part of. I can dig down further to try to figure out what the issue is.